libs/map-editor/src/types.ts

```diff
@@ -218,6 +216,12 @@ export const EntityCollectionRaw = z.object({
     version: z.string().optional(),
 });
 
+export const InputTagOption = z.object({
+    value: z.string(),
+    label: z.string(),
+    created: z.boolean().optional(),
+});
+
```

Comment:
You can move this to "play" and use a type or interface instead because you never use "parse" or "safeParse" on this object.